### DISCUSSION TITLE
支持类似bookmarks的功能吗？比如在代码中跳转了很多次，但是每次都一步一步返回很繁琐。

### DISCUSSION THREAD

**Discussion by Robin329** [ayamir/nvimdots#951] [Q&A] [answered]
### Feature description

代码中跳转是非常频繁的，但是我们某一刻只需要关注其中一个函数，将他标记为一个书签，以便后续直接跳转到该书签。之前vim是有类似的插件，但是neovim是否有对应改进的插件呢？

**Comment by Jint-lzxy** [comment]
This should be in discussions, transferred.

**Comment by Jint-lzxy** [comment, marked as answer]
There're `:h mark-motions` Marks, `:h quickfix` Qf-commands, and `:h jumplist` Jumplists. All of them are builtin.

**Comment by fecet** [comment]
Use `<C-t>` and `<C-o>`

**Comment by kang8** [comment]
Try `:Telescope jumplist` using telescope search jumplist.
